Plugin.php

```diff
@@ -153,5 +153,10 @@ public function register()
         CombineAssets::registerCallback(function ($combiner) {
             $combiner->registerBundle('$/rainlab/builder/assets/js/build.js');
         });
+
+        /**
+         * Add provider for "timestamp" type for MySQL and SQLite
+         */
+        $this->app->register(\MarkTopper\DoctrineDBALTimestampType\Laravel5ServiceProvider::class);
```

Comment:
Is this going to break when L6 is released?

Reply:
@LukeTowers I don't think so - the timestamp field still isn't available in Doctrine with L6, but I'll give it a test to make sure.

Reply:
I moreso meant that we're referencing a service provider specifically made for Laravel 5 so I'd be concerned about composer versions of the package no longer including it for a Laravel 6 install.

Reply:
@LukeTowers I've tested this with L6 and it works fine. The `composer.json` for this library does not specify a Laravel version, it only requires `doctrine/dbal`, and its restriction is compatible with Laravel 6. The service provider also only uses methods from Doctrine and does not interact with Laravel at all beyond latching to its service provider boot process.